subsys/bluetooth/controller/hci_internal.c

```diff
@@ -405,6 +405,7 @@ static void le_supported_features(sdc_hci_le_le_features_t *features)
 
 	features->le_encryption = 1;
 	features->extended_reject_indication = 1;
+	features->slave_initiated_features_exchange = 1;
```

Comment:
```suggestion
#if CONFIG_BT_PERIPHERAL
	features->slave_initiated_features_exchange = 1;
#endif
```

Reply:
These are controller supported features, not configured per role right?

Reply:
both sides must support the feature for it to be usable. I.e. master also must declare that it is able to respond to it, right?

Reply:
Ah, yes, @seko-nordic , you are right. This change should be reverted.